didadata/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = '0.4.0'
+__version__ = '0.5.0'
```

Comment:
major release as we drop a lot of support

Reply:
Okidoki